src/reconcile/rec_row.c

```diff
@@ -860,12 +860,13 @@ __wt_rec_row_leaf(
                 }
 
                 /*
-                 * If we're removing a key, also remove the history store contents associated with
-                 * that key. Even if we fail reconciliation after this point, we're safe to do this.
-                 * The history store content must be obsolete in order for us to consider removing
-                 * the key.
+                 * If we're removing a key with durable timestamp of tombstone is NONE, also remove
```

Comment:
Typo: I think it's better worded as `If we're removing a key due to a tombstone with a durable timestamp of "none"`.

Reply:
Thanks. Fixed using the suggested sentence.